### ISSUE TITLE
Handling event when Image Selection in Cancelled from Album

### ISSUE THREAD

**Issue by Mr-Anonymous** [dmcBig/cordova-plugin-mediaPicker#6]
I noticed that when you go to the Album and then press Cancel without selecting an image, an error is not returned when no image is obtained. This is the code I have:

```
var args={
                  'selectMode': 100,
                  'maxSelectCount': 40,
              };

MediaPicker.getMedias(args, function(results) {
   console.log(results);
}, function (error) {
    console.log('Error: ' + error);
    alert("Unable to obtain pictures");
}); 
```

In the above, when the image selection is cancelled, I thought I should get the `Unable to obtain pictures` alert but that doesn't work. Neither the success nor the error method is triggered when cancelled. This is the same in both iOS and Android App. How can I handle this event?

This is my ionic info:

```
cli packages: (/usr/local/lib/node_modules)

    @ionic/cli-utils  : 1.19.2
    ionic (Ionic CLI) : 3.20.0

global packages:

    cordova (Cordova CLI) : 8.0.0 

local packages:

    Cordova Platforms : android 7.0.0 ios 4.5.4
    Ionic Framework   : ionic1 1.3.3

System:

    ios-deploy : 1.9.2 
    ios-sim    : 5.0.13 
    Node       : v9.3.0
    npm        : 5.7.1 
    OS         : macOS High Sierra
    Xcode      : Xcode 9.2 Build version 9C40b 

Environment Variables:

    ANDROID_HOME : not set

Misc:

    backend : pro
```

Thanks,
Neel.

**Comment by dmcBig** [comment]
Currently not supported, I will add

**Comment by Mr-Anonymous** [comment]
May I ask do you have a rough ETA on when you might be able to add this please? I wrap the image picker inside a service that has to return a promise. Even if the cancel event can be caught inside success in case its not thrown as an error, it should be good enough. At the moment, these promises remains unresolved. Thank you for your support.

**Comment by Mr-Anonymous** [comment]
Hi @dmcBig I updated the plugin today and I noticed that when the Media Picker is cancelled, it is returning an empty result. This is perfect since now I can resolve the promise easily by checking if the `result.length` is empty or not. Thank you so much for adding this.

I love this plugin, it works really well and I have added this plugin in my production app now. Thanks heaps and keep up the awesome work.
